### PR TITLE
Seperate MaxQuorums

### PR DESCRIPTION
**Pull request by heidihoward**
Fixing a bug introduced during the refactoring of `MaxQuorum` https://github.com/heidihoward/pirateship-tla/commit/035b5aebef0b0685ef71af7b8b93600d6f9adb87#diff-bb74f8203fb9704ca17cbe4e4b2335f443a4659bed373276a16ccb626a989b81L257. These two calculations were using different quorums (CQ for crash quorum and BQ for Byzantine quorum), when they were merged they were both switched to BQ

CC @lemmy

### CODE REVIEW

**Comment by lemmy** [comment]
Apologies!

File: pirateship.tla [hunk 1]
```diff
@@ -184,7 +184,15 @@ HighestQCOverQC(l) ==
 Max2(a,b) == IF a > b THEN a ELSE b
 Min2(a,b) == IF a < b THEN a ELSE b
 
-MaxQuorum(l, m, default) == 
+MaxCrashQuorum(l, m, default) ==
```
Comment (lemmy) [inline pirateship.tla:187]: You could also make the quorum a parameter of `MaxCrashQuorum`.

```tla
MaxQuorum(Q, l, m, default) == 
    LET RECURSIVE RMaxQuorum(_)
        RMaxQuorum(i) ==
            IF i = default THEN default
            ELSE IF \E q \in Q: \A n \in q: m[n] >= i
                 THEN i ELSE RMaxQuorum(i-1)
    IN RMaxQuorum(Len(l))
```

**Review by lemmy — APPROVED** [review]
-

**Comment by lemmy** [comment]
The definitions of `CQ` and `BQ` are identical:

https://github.com/heidihoward/pirateship-tla/blob/9a611a418a5d1e4025f6c28e5fc9b3034a28fdc7/pirateship.tla#L68-L71

**Comment by heidihoward** [comment]
You change them when you change the number of replicas. N=4 is a special case where they are the same

**Comment by lemmy** [comment]
I never touched `CQ` and `BQ`:

https://github.com/heidihoward/pirateship-tla/commit/6a6c86212954b00e8633e4d018b4a729d53666f1#diff-bb74f8203fb9704ca17cbe4e4b2335f443a4659bed373276a16ccb626a989b81R49-R53

**Comment by heidihoward** [comment]
Sorry I meant to say "You change them" not "You changed them" as in you can change them if you want to vary the number of nodes

**Pull request merged**